### PR TITLE
Database setup

### PR DESCRIPTION
**Pull request by matthewdking**
relates #32 #30 

- creates database connection
- creates database build

### CODE REVIEW

File: database/db_build.sql [hunk 1]
```diff
@@ -0,0 +1,59 @@
+BEGIN;
+
+DROP TABLE IF EXISTS users, careers, admins, universities, users_careers, universities_careers cascade;
+
+CREATE TABLE users(
+  user_id               SERIAL    PRIMARY KEY,
```
Comment (finnhodgkin) [inline database/db_build.sql]: I would leave all the primary keys as just `id`. The only tables with a different name for `id`s should be the relational tables. `user_careers` and university_careers

**Comment by finnhodgkin** [comment]
Having every language `not-null` puts quite a lot of burden on the person filling in the career form. I don't mind it, but might be worth checking with the PO at some point.

**Comment by matthewdking** [comment]
@finnhodgkin I thought the same but without having all the fields filled in what would we show on the cards?

**Comment by finnhodgkin** [comment]
Yeah I'm not sure. As long as we make the careers form editable it should be fine. Then anyone who doesn't speak every language can come back with a pal later.

**Comment by finnhodgkin** [comment]
LGTM merging :sushi:

**Pull request merged**